qiskit/providers/aer/backends/backend_utils.py

```diff
@@ -16,7 +16,7 @@
 """
 import os
 from math import log2
-from qiskit.utils import local_hardware_info
+from qiskit.util import local_hardware_info
```

Comment:
This is reverting #1382 by accident.
```suggestion
from qiskit.utils import local_hardware_info
```